web/frontend/components/ChecksTable/CheckResultIcon.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const PASSING = 'passing';
```

Comment:
We have the SKIPPED state as well (which uses the `fiber_manual_record` type of icon).
Maybe useful to know, if we want to add tooltips or similar at some point

Reply:
Yeah, I defaulted to it, thanks for discussing! Do you prefer another default?

Reply:
Yes, add other variable as `SKIPPED = 'skipped'`. Even that it will use the same icon, let's have it in the code to have a better picture